testing/test_cluster.py

```diff
@@ -24,19 +25,19 @@ def createTestValue(self):
     def make_cluster(self, num_nodes):
         self.assertGreaterEqual(num_nodes, 3)
 
-        ensemble = {i: '10.0.1.{}'.format(i) for i in xrange(1, num_nodes + 1)}
-        zk_servers = ctu.ensemble_servers_string(ensemble)
-        external_storage = ctu.external_storage_string(ensemble)
+        self.ensemble = {i: '10.0.1.{}'.format(i) for i in xrange(1, num_nodes + 1)}
+        zk_servers = ctu.ensemble_servers_string(self.ensemble)
+        external_storage = 'zk:' + ctu.external_storage_string(self.ensemble)
```

Comment:
I understand that, with your change, some calls need the 'zk:' prefix, and other calls do not. but moving the 'zk:' string over to this file for concatenation feels a bit tacky. Wouldn't it be cleaner to simply add a boolean parameter use_zk_prefix for the ctu.external_storage_string() method?